### PR TITLE
453 apply maximize direction

### PR DESCRIPTION
**Pull request by r-soga**

### CODE REVIEW

File: examples/pruning/chainermn_integration.py [hunk 1]
```diff
@@ -80,6 +80,11 @@ def objective(trial, comm):
     trainer.extend(
         optuna.integration.ChainerPruningExtension(trial, 'validation/main/loss',
```
Comment (sile) [inline examples/pruning/chainermn_integration.py]: It seems that `validation/main/loss` needs to be modified.

File: examples/pruning/lightgbm_integration.py [hunk 2]
```diff
@@ -43,18 +43,21 @@ def objective(trial):
...
-    return 1.0 - accuracy
+    return accuracy
 
 
 if __name__ == '__main__':
-    study = optuna.create_study(pruner=optuna.pruners.MedianPruner(n_warmup_steps=10))
+    study = optuna.create_study(
+        pruner=optuna.pruners.MedianPruner(
+            n_warmup_steps=10),
+        direction='maximize')
```
Comment (c-bata) [inline examples/pruning/lightgbm_integration.py]: It looks indents are not aligned.

File: examples/pruning/xgboost_integration.py [hunk 3]
```diff
@@ -49,15 +50,18 @@ def objective(trial):
...
-    return 1.0 - accuracy
+    return accuracy
 
 
 if __name__ == '__main__':
-    study = optuna.create_study(pruner=optuna.pruners.MedianPruner(n_warmup_steps=5))
+    study = optuna.create_study(
+        pruner=optuna.pruners.MedianPruner(
+            n_warmup_steps=5),
+        direction='maximize')
```
Comment (c-bata) [inline examples/pruning/xgboost_integration.py]: ditto.

File: examples/pruning/chainermn_integration.py [hunk 4]
```diff
@@ -99,15 +99,19 @@ def objective(trial, comm):
...
     # Please make sure common study and storage are shared among nodes.
     study_name = sys.argv[1]
     storage_url = sys.argv[2]
 
-    study = optuna.load_study(study_name, storage_url, pruner=optuna.pruners.MedianPruner())
+    study = optuna.load_study(
+        study_name,
+        storage_url,
+        pruner=optuna.pruners.MedianPruner(),
+        direction='maximize')
```
Comment (toshihikoyanase) [inline examples/pruning/chainermn_integration.py]: Please remove `direction='maximize'` because `optuna.load_study` does not have `direction` argument.
In addition to that, please add `--direction maximize` to [L11](https://github.com/pfnet/optuna/pull/526/files#diff-2d1b3387c842d67fa53c28335838bf46R11) as follows:

```
    $ STUDY_NAME=`optuna create-study --storage $STORAGE_URL --direction maximize`
```

**Comment by toshihikoyanase** [comment]
@r-soga 
Thank you for reflecting my comments.
Some CI checks failed, but the issue was solved by this PR (https://github.com/pfnet/optuna/pull/537).
Please merge this branch with master.
